nextjs/sentry.client.config.ts

```diff
@@ -5,15 +5,22 @@
 import * as Sentry from "@sentry/nextjs";
 
 Sentry.init({
-  dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
+  dsn: process.env.SENTRY_DSN,
```

Comment:
It looks like this will prevent the browser from accessing the SENTRY_DSN, so we won't get frontend error reporting anymore. If I have that right, then we should NEXT_PUBLIC_ for the Sentry DSN.

If you're concerned about receiving events from non-Mapped URLs, let's just set a whitelist in Sentry.